### PR TITLE
ITM-81: Use configurable host/port in server

### PR DESCRIPTION
**Pull request by wgossard**
Changed Hardcoded localhost and ports to use ENV vars.
All have original defaults

### CODE REVIEW

File: swagger_server/itm/itm_scenario_session.py
Comment (nextcen-dgemoets) [inline swagger_server/itm/itm_scenario_session.py:64]: So I know I asked you to update the mongo call to use TA3_HOSTNAME, but does that make sense?  TA3_HOSTNAME is where the TA3 server is running, and the database is only accessed by the TA3 server, which means that localhost should always work, no?   Now, we can certainly add configuration to be able to run the db on a different host and/or port, but that would be separate config that I don't think we need at the moment.  Thoughts?

**Review by nextcen-dgemoets — COMMENTED** [review]
Other than the one comment regarding the DB, everything looks good here from the non-AWS, non-Docker standpoint.  That is, the port changes.  @KageRyu7 will need to verify multi-machine functionality in AWS.

File: swagger_server/itm/itm_scenario_session.py
Comment (brianpippin) [reply swagger_server/itm/itm_scenario_session.py:64]: I believe when running this on AWS, mongo would always be accessible at localhost but it doesn't hurt having it customizable should we need to switch things up in the future.

Comment (nextcen-dgemoets) [reply swagger_server/itm/itm_scenario_session.py:64]: @brianpippin  Do you want both host and port customizable by envvar (e.g., DB_HOST, DB_PORT), or just one or the other?

Comment (brianpippin) [reply swagger_server/itm/itm_scenario_session.py:64]: Not too worried about the port, mongo almost always runs at 27017 by default, so unless we start getting into multiple mongo dbs, I figure we can cross that bridge later

**Review by nextcen-dgemoets — APPROVED** [review]
Approved, pending testing on AWS.  Depending on timing, be sure to pull the latest from development before merging back into development.

**Pull request merged**
